diff --git a/Makefile.in b/Makefile.in
index 71f59cf..acb335d 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -81,17 +81,6 @@ endif
 
 # change ownership unless we're building an rpm
 ifeq ($(RPM_OS)$(DARWIN),)
-  SETOWNER=--owner iguanair --group iguanair
-install: adduser
-.PHONY: adduser
-adduser:
-	/usr/sbin/useradd -u 213 -c "Iguanaworks IR Daemon" -d / -s /sbin/nologin iguanair 2>/dev/null || true
-
-uninstall: deluser
-.PHONY: deluser
-deluser:
-	/usr/sbin/userdel iguanair || true
-else
   SETOWNER=
 endif
 
